applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -182,10 +182,10 @@ static int adv_create(void)
 
 #if (CONFIG_AURACAST)
 	NET_BUF_SIMPLE_DEFINE(pba_buf, BT_UUID_SIZE_16 + 2);
-	struct bt_data ext_ad[3];
+	struct bt_data ext_ad[4];
```

Comment:
Why do we use CONFIG_AURACAST here? This seems like a define that should only be set when the Auracast requirements are met, and I would expect `ext_ad[4];` even for standard broadcast?

Reply:
The Auracast is used because the fourth spot is for the Public Broadcast Announcement (PBA), might not want to have that for all broadcasters